### ISSUE TITLE
compilation error not caught by linter

### ISSUE THREAD

**Issue by ns-gsa** [github/super-linter#3019]
The below code passed linting, whereas groovy compiler clearly throws error

`df greeting = "Hello"`

groovyc
```
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
def.groovy: 1: unable to resolve class df
 @ line 1, column 4.
   df greeting = "Hello"
      ^
```

super-linter
```
2022-06-13 01:21:38 [INFO]   Linting [GROOVY] files...
2022-06-13 01:21:38 [INFO]   ----------------------------------------------
2022-06-13 01:21:38 [INFO]   ----------------------------------------------
2022-06-13 01:21:38 [INFO]   ---------------------------
2022-06-13 01:21:38 [INFO]   File:[def.groovy]
2022-06-13 01:21:42 [INFO]    - File:[def.groovy] was linted with [npm-groovy-lint] successfully
2022-06-13 01:21:42 [INFO]      - Command output:

```
